linux_docker_resources/rti_web_binaries_install_script.py

```diff
@@ -36,11 +36,18 @@ def install_connext(installer_path, install_directory):
         child.sendline(install_directory)
         child.expect_exact('Do you want to continue? [Y/n]:')
         child.sendline('y')
-        result_index = child.expect_exact([
-            'Create an RTI Launcher shortcut on the Desktop [y/N]: ', pexpect.EOF], timeout=120)
-        if result_index == 0:
-            child.sendline('n')
-            child.expect(pexpect.EOF)
+        while True:
+            result_index = child.expect_exact([
+                pexpect.EOF,
+                'Disable copying of examples to rti_workspace [y/N]: ',
+                'Create an RTI Launcher shortcut on the Desktop [y/N]: ',
+                ], timeout=120)
+            if result_index == 0:
+                return
+            elif result_index == 1:
+                child.sendline('y')
+            elif result_index == 2:
+                child.sendline('n')
```

Comment:
I assume the index should never be not 0, 1, or 2? For safety I would propose to still add:

```
else:
    assert False, 'Unexpected result_index: %d' % result_index
```

Reply:
it'll return an index in the list or timeout (which is caught below), but there's no harm in adding it. How come not raise though? (not that we intend to run this optimised, but who knows.)

Reply:
> How come not raise though?

That is exactly what the `assert` does.

Reply:
not if run with -O, right? Again it's "not that we intend to run this optimised, but who knows." :smile:

Reply:
=> what I meant was that today we don't, but for some unrelated reason in the future we might set `PYTHONOPTIMIZE` or something